ietf/meeting/management/commands/populate_attended.py

```diff
@@ -0,0 +1,76 @@
+# Copyright The IETF Trust 2022, All Rights Reserved
+
+import debug # pyflakes: ignore
+
+from tqdm import tqdm
+
+from django.core.management.base import BaseCommand
+
+#from ietf.meeting.models import Meeting, Session
+from ietf.meeting.models import Session
+#from ietf.meeting.helpers import get_schedule
```

Comment:
Are these imports likely enough to come back / non-obvious enough to warrant keeping around as comments?

Reply:
removed them